src/item_pocket.h

```diff
@@ -381,6 +385,17 @@ class item_pocket
 
         favorite_settings settings;
 
+        // Pocket presets functions
+        void serialize_presets( JsonOut &json ) const;
+        void deserialize_presets( const JsonArray &ja );
+        void load_presets();
+        void add_preset( const item_pocket::favorite_settings &preset );
+        void save_presets() const;
+        std::vector<item_pocket::favorite_settings>::iterator find_preset( const std::string &s );
+        bool has_preset( const std::string &s );
+        void delete_preset( std::vector<item_pocket::favorite_settings>::iterator iter );
+        std::vector<item_pocket::favorite_settings> pocket_presets;
```

Comment:
To my understand this is supposed to be a global value rather than a per-pocket value? If so you should make it static to avoid wasting memory and triggering the clang-tidy warning.